### PR TITLE
Fixed the code so that the example code works properly

### PR DESCRIPTION
**Pull request by phanker**
1. nft objects want image_url to display image information properly, not img_url.
2. should be using transfer::publish_transfer method instead of directly using transfer(object,sender).
3. Improve the mint method to make the contract mint capable

### CODE REVIEW

**Review by damirka — APPROVED** [review]

File: docs/content/standards/display.mdx [hunk 1]
```diff
@@ -107,14 +107,15 @@ module examples::my_hero {
...
+        transfer::public_transfer(display, sender(ctx));
     }
 
     /// Anyone can mint their `Hero`!
-    public fun mint(name: String, img_url: String, ctx: &mut TxContext): Hero {
+    public fun mint(name: String, img_url: String, ctx: &mut TxContext) {
         let id = object::new(ctx);
-        Hero { id, name, img_url }
+        let hero = Hero { id, name, img_url };
+        transfer::public_transfer(hero, sender(ctx));
```
Comment (damirka) [inline docs/content/standards/display.mdx:118]: I wouldn't change this signature. This is how we do things today - return so the sender can decide whether to transfer or not.

**Review by damirka — CHANGES_REQUESTED** [review]
Requesting changes to make sure the 'mint' function signature is not touched.

**Comment by phanker** [comment]
> Requesting changes to make sure the 'mint' function signature is not touched.

I agree with this, I just made a regression of the mint function. But I don't think it's too beginner friendly, because this 'mint' method allows anyone to call it, and I believe it's intended to mint a nft to its own account, not only create, or transfer

**Review by damirka — APPROVED** [review]

**Comment by damirka** [comment]
Thank you for this!

**Comment by stefan-mysten** [comment]
Thanks @phanker for the PR!

**Pull request merged**